### ISSUE TITLE
Could you please share the pre-trained model weights?

### ISSUE THREAD

**Issue by easonyang1996** [jcboyd/cdpath21-gan#1]
Could you please share the pre-trained model weights? I want to try the pre-trained ResNet18 on the NCT-CRC dataset.

**Comment by jcboyd** [comment]
Hi, all models are now available via the colab links at the top of the README.

I should point out that the CRC model has been retrained, and therefore does not have the same weights as in the paper. As a result, performance is slightly different (slightly better FID, slightly worse balanced accuracy).
